github.properties

```diff
@@ -0,0 +1,2 @@
+gpr.usr = 49334352
```

Comment:
What is this user name and password? Can you explain ?

Reply:
It is used for Liquid Pager which is used in onboarding fragment.